src/parsing.jl

```diff
@@ -67,6 +67,54 @@ function composed_or_symbol(x::Expr)
         all(composed_or_symbol, x.args[2:end])
 end
 
+is_call(x) = false
+is_call(x::Expr) = x.head == :call
+
+is_nested_fun(x) = false
+function is_nested_fun(x::Expr)
+    x.head == :call &&
+    length(x.args) == 2 &&
+    is_call(x.args[2]) &&
+    # AsTable(:x) or `$(:x)`
+    get_column_expr(x.args[2]) == nothing
```

Comment:
I understand you catch here cases where user passes things like `Cols(...)` or similar?

Reply:
Actually `Cols` isn't recognized. It's just `AsTable` and `$(...)`. `AsTable` is the only way to do multiple column selection at the moment.